### DISCUSSION TITLE
Use cases for the bot

### DISCUSSION THREAD

**Discussion by Steffo99** [Steffo99/greed#65] [General]
Hi everyone!

I'd like to know if anyone is currently using Greed in a production environment!

If you do, please post a reply at this issue describing where and how you use it :)

**Comment by urba1n** [comment]
Thinking of hosting a few free bots to help local businesses to sell their products during closing involved by covid. But paiement/wallet seems too complicated so pay cash (or anyway, but at delivery) could be nice. Did you already reconsider this feature ?

**Comment by Steffo99** [comment]
@urba1n What?

**Comment by yayitazale** [comment]
We tried it in test environment for a local craft beer shop but for the owner it was too much complicated (because of using credits instead of direct payments and some other things).

The base of this project is fantastic but it need some mayor changes to be usable on any kind of product shop. 

Anyway, thanks for your work.

**Reply by Steffo99** [reply]
Many many thanks for the feedback! :smile:
